modules/ROOT/pages/kubernetes/quickstart-analytics-cluster.adoc

```diff
@@ -0,0 +1,168 @@
+:description: How to deploy a Neo4j cluster to a cloud or a local Kubernetes cluster using Neo4j Helm chart.
+[role=enterprise-edition]
+[[quick-start-analytic-cluster]]
+= Quickstart: Deploy a Neo4j cluster for analytic queries
+
+_The feature is available in the Neo4j Helm chart from version 5.14._
+
+This quickstart shows how to configure and deploy a special Neo4j cluster without fault tolerance to support analytic queries.
+Information on using Neo4j’s graph data science library in a cluster can be found in the link:https://neo4j.com/docs/graph-data-science/current/[Neo4j Graph Data Science library documentation].
+
+The cluster is deployed to a cloud or a local Kubernetes cluster using the Neo4j Helm chart.
+Because the GDS library does not support fault tolerance, you can deploy a cluster with one primary server for the transaction workload and an N number of secondary servers for the GDS analytics.
+
+== Prerequisites
+
+Before you can deploy a Neo4j cluster on Kubernetes, you need to have:
+
+* A Kubernetes cluster running and the `kubectl` command-line tool installed and configured to communicate with your cluster.
+For more information, see link:xref:kubernetes/quickstart-cluster/prerequisites.adoc[Quickstart: Deploy a cluster -> Prerequisites].
+* A valid license for Neo4j Enterprise Edition.
+For more information, see xref:/kubernetes/plugins.adoc#install-gds-ee-bloom[Install GDS Enterprise Edition (EE) and Bloom plugins].
+* The xref:kubernetes/helm-charts-setup.adoc[latest version of the Neo4j Helm chart repository].
+* (Optional) A valid license for GDS Enterprise Edition.
+To install a licensed plugin, you must provide the license files in a Kubernetes secret.
+For more information, see xref:/kubernetes/plugins.adoc#install-gds-ee-bloom[Install GDS Enterprise Edition (EE) and Bloom plugins].
+
+== Create a value YAML file for each type of server
+
+To set up a Neo4j cluster for analytic queries, you need to create a value YAML file for each type of server, primary and secondary.
+For example:
+
+[.tabbed-example]
+=====
+[.include-with-primary]
+======
+
+Create a value YAML file for the primary server, for example, _primary-value.yaml_:
+
+[source, yaml]
+----
+neo4j:
+  name: analytics-cluster
+  acceptLicenseAgreement: "yes"
+  edition: enterprise
+  password: my-password
+volumes:
+  data:
+    mode: defaultStorageClass
+
+# Disable the Neo4j load balancer and enable the internal service so that the servers can access each other:
+services:
+  neo4j:
+    enabled: false
+  internals:
+    enabled: true
+
+# Enable the analytics cluster and set the type to primary:
+analytics:
+  enabled: true
+  type: primary
```

Comment:
this is wrong , the correct syntax is this. Please refer to the demo video again.
```
analytics:
  enabled: true
  type: 
    name: primary
```